setup.cfg

```
@@ -36,6 +36,7 @@ install_requires =
     numpy
     pint
     zarr
+    nexgen
```

This will ensure you get the necessary minimum version of Nexgen.
```suggestion
    nexgen >= 0.4.3
```